pkgs/applications/science/math/R/default.nix

```diff
@@ -49,10 +52,22 @@ stdenv.mkDerivation rec {
       LDFLAGS="-L${gfortran.cc}/lib"
       RANLIB=$(type -p ranlib)
       R_SHELL="${stdenv.shell}"
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+      --without-tcltk
+      --disable-R-framework
+      CC="clang"
```

Comment:
I only know that the build failed without those settings. /cc @pikajude @copumpkin